test_scripts/Polices/Policy_Table_Update/010_ATF_P_TC_HMI_sends_GetURLs_one_app_registered.lua

```diff
@@ -0,0 +1,77 @@
+---------------------------------------------------------------------------------------------
+-- Requirements summary:
+-- In case HMI sends GetURLs and at least one app is registered SDL must return only default url and url related to registered app
```

Comment:
Name of requirement currently  is [PolicyTableUpdate] Policy Manager responds on GetURLs from HMI please check